grazel-gradle-plugin/src/main/kotlin/com/grab/grazel/gradle/variant/ConfigurationParsingVariant.kt

```diff
@@ -106,6 +108,8 @@ interface ConfigurationParsingVariant<T> : Variant<T> {
                 Test -> configName == "test${basePattern}${onlyConfig}$dm".toLowerCase()
                     || configName == "test${basePattern}Implementation$dm".toLowerCase()
 
+                Lint -> configName == "lintChecks".toLowerCase()
```

Comment:
Could you consider adding a test case to cover this setup? Currently other variant types are setup and checked if parses correctly, can do that same here..

https://github.com/grab/grazel/blob/master/grazel-gradle-plugin/src/test/kotlin/com/grab/grazel/gradle/variant/VariantTest.kt